ci-operator/step-registry/storage/conf/storageclass/pre-set-default-azure-file-cross-subscription/storage-conf-storageclass-pre-set-default-azure-file-cross-subscription-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Abort the job if the cross subscription is not created
+if [[ ! -f "${SHARED_DIR}/resourcegroup_cross-sub" ]]; then
+  echo "Error: The cross subscription is not created, exit!"
+  exit 1
```

Comment:
maybe "exit 0" is more safe to avoid noise when adding this step into existing workflow.

Reply:
In the CI, we expect to create the azurefile-csi storage class (the same name but contains the sub/rg info) which will be used in the test, we have to fail the step if it is not created. Otherwise, when we still use the default azurefile-csi storage class, all cases will pass but without any cross-subscription coverage, so I have to "exit 1" here to emphasize it.

This step will not be re-used too much, I guess it is for storage cross-subscription only. And usually it will not fall into this "if" section :)

Reply:
same as https://github.com/openshift/release/pull/64263#discussion_r2068302401